ngboost/tests/test_distns.py

```diff
@@ -38,22 +38,42 @@ def product_list(*its: Iterable) -> List:
 
 @pytest.mark.slow
 @pytest.mark.parametrize(
-    ["dist", "score", "learner"],
-    product_list(
```

Comment:
can we get rid of `product_list`?

Reply:
good idea, lemme purge the rest of the test base

Reply:
if you wanna do it in a seperate PR, feel free to - am just rambling :D